### PR TITLE
WIP: DO NOT MERGE: Apache Arrow 2022-04 board report

### PR DESCRIPTION
**Pull request by kou**
This is a draft of Apache Arrow 2022-04 board report. Committers can
push directly to this branch. Others can leave suggestions.

This is never merged to apache/arrow. This will be reported by https://reporter.apache.org/wizard/?arrow .

### CODE REVIEW

File: board-report-2022-04.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- RS-8.0.0 was released on 2022-01-25.
+- RS-7.0.0 was released on 2022-01-12.
+
+## Community Health:
+
+- We've released a new version of Apache Arrow Julia after it was re-donated.
+- We'll start IP clearance for Apache Arrow Flight JDBC driver donation.
+- We may register Apache Arrow as a Digital Public Good like Apache
+  Fineract: https://digitalpublicgoods.net/registry/apache-fineract.html
+- We'll rename the "master" branch to "main".
```
Comment (nealrichardson) [inline board-report-2022-04.md:46]: Maybe add a reference to the umbrella issue for this (ARROW-15689) and acknowledge that we'll be asking INFRA for support on this when we're ready.

**Review by nealrichardson — APPROVED** [review]
One minor suggestion but LGTM, thank you for putting this together!

File: board-report-2022-04.md (hunk 1 above)
Comment (kou) [reply board-report-2022-04.md]: Thanks for the suggestion!
I've applied it.

File: board-report-2022-04.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- RS-DF-7.0.0 was released on 2022-02-17.
+- RS-9.0.2 was released on 2022-02-13.
+- 7.0.0 was released on 2022-02-03.
+- RS-8.0.0 was released on 2022-01-25.
+- RS-7.0.0 was released on 2022-01-12.
+
+## Community Health:
+
+- We've released a new version of Apache Arrow Julia after it was re-donated.
+- We'll start IP clearance for Apache Arrow Flight JDBC driver donation.
```
Comment (lidavidm) [inline board-report-2022-04.md:43]:
```suggestion
- We'll start IP clearance for Apache Arrow Flight SQL JDBC driver donation.
```

Just a nit. Thanks for putting this together!

Comment (kou) [reply board-report-2022-04.md]: Good catch!

**Comment by kou** [comment]
I've submitted this.

**Pull request closed without merging**